light_controller/ir_toy/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+__author__ = 'akm'
```

Comment:
Should this be here? If yes, make sure to add author for `light_controller/_init_/py` and use `"` for strings